blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/Constants.java

```diff
@@ -66,7 +65,7 @@
             throw new RuntimeException("Invalid FullResSerializer.CUR_VERSION. Please increment until this exception does not happen.");
     }
 
-    public static final Charset DEFAULT_CHARSET = CharsetUtil.UTF_8;
+    public static final Charset DEFAULT_CHARSET = Charset.forName("UTF-8");
```

Comment:
I think this still exists in 4.0 https://netty.io/4.0/api/io/netty/util/CharsetUtil.html  Looks like you use it later https://github.com/rackerlabs/blueflood/pull/726/files#diff-02100b1d3f0794dc57e4870d8bb80873R63

Was there a different reason to remove it?